.github/workflows/admin-semantic_release-pr.yml

```diff
@@ -0,0 +1,106 @@
+---
+on:
+  pull_request:
+    branches:
+      - main
+      - production
+      - next
```

Comment:
NB: Is this strictly necessary? 
- Clearly, PRs opened from a `feat` branch into `main` should have a semantic release changelog preview
- Clearly, any release PR, opened from `next` and into `production` should have a release changelog preview

But do we ever actually open PRs into `next? 

(Asking mainly for my own understanding)

Reply:
not strictly, but it catches the scenario where `next` is manually created through GH (prior to a PR). It also catches the case (bad practice, but it happens), where a last feature gets shoehorned into a release